### PR TITLE
Casting call info

### LINKED ISSUE #727: Collecting info and headshots (SJP, CCP)
Collecting info and headshots on NNT people from [castingcallpro](http://www.castingcallpro.com/uk/search/actors?name=&keywords=nottingham+new+theatre&sex=&spotlight=&imdb=&aftra=&equity=&sag=&showreel=&voicereel=&headshot_360=&agent_represented=&heightfrom=1&heightto=300&height=&weightfrom=1&weightto=200&weight=&site_uid=1&end_lastlogin=2017-03-03+14%3A00%3A35&agefrom=16&ageto=99) and [stageprojobs](http://www.stagejobspro.com/uk/search/theatre-professionals?name=&keywords=nottingham+new+theatre&sex=&spotlight=&imdb=&aftra=&equity=&sag=&showreel=&voicereel=&headshot_360=&agent_represented=&heightfrom=1&heightto=300&height=&weightfrom=1&weightto=200&weight=&site_uid=23&end_lastlogin=2017-03-03+14%3A04%3A37&agefrom=18&ageto=99)

---

- [ ] http://www.castingcallpro.com/uk/actor/profile/tom-redican is https://history.newtheatre.org.uk/people/tom_walsh/

### PR DESCRIPTION
**Pull request by nhyty**
Close #727 for now

### CODE REVIEW

**Comment by BenWoodford76** [comment]
Build Failing because of this:

 'Liquid Exception: undefined method `[]' for nil:NilClass in /_layouts/person.html'

**Comment by BenWoodford76** [comment]
Do the links need the quotation marks around them?

**Comment by BenWoodford76** [comment]
Also, Some Fellow headshots in this branch :) #666

**Comment by nhyty** [comment]
Any chance you can look at this will?

File: _people/chris_walters.md [hunk 1]
```diff
@@ -8,4 +8,6 @@ submitted: false
 links:
   - type: CastingCallPro
     href: "https://www.castingcallpro.com/uk/actor/profile/christopher-walters-1"
+  - type: Portfolio
```
Comment (wjdp) [inline _people/chris_walters.md]: Sorry for being a pain, think this should just be `SoundCloud`

**Pull request merged**